targets/TARGET_Samsung/TARGET_SIDK_S5JS100/.mbedignore

```diff
@@ -0,0 +1 @@
+serial_api_old.c
```

Comment:
Is this still needed ? this ignore should  be removed along with t his _old file?

Reply:
The .mbedignore file is deleted.